### PR TITLE
Fix FinalizedDataModelIsMarkedAsFinalizedInXMLSpec

### PR DESCRIPTION
**Pull request by vijayshukla30**
Download the XML of the finalized data model and check the downloaded XML has the finalized data model name. Changes are three files:
**GebConfig**: Added the configuration to download the file and save it a location  
**DataModelPage**: Added the download link.  
**FinalizedDataModelIsMarkedAsFinalizedInXMLSpec**: Testing file

### CODE REVIEW

File: ModelCatalogueCorePluginTestApp/test/functional/GebConfig.groovy [hunk 1]
```diff
@@ -15,7 +17,22 @@ environments {
     // run via “./gradlew chromeTest”
     // See: http://code.google.com/p/selenium/wiki/ChromeDriver
     chrome {
-        driver = { new ChromeDriver() }
+        String downloadFilepath = "/home/vijay/"
```
Comment (sdelamo) [inline ModelCatalogueCorePluginTestApp/test/functional/GebConfig.groovy]: @Vendetta30 could you wrap this code with : 

```
if ( System.getProperty('downloadFilepath') ) {
String downloadFilePath = System.getProperty('downloadFilepath')
.........
} else {
  driver = { new ChromeDriver() }		
}
```

File: ModelCatalogueCorePluginTestApp/test/functional/org/modelcatalogue/core/remoteTesting/FinalizedDataModelIsMarkedAsFinalizedInXMLSpec.groovy [hunk 2]
```diff
@@ -23,6 +24,10 @@ import spock.lang.Title
 class FinalizedDataModelIsMarkedAsFinalizedInXMLSpec extends GebSpec {
 
     def 'Examine that finalized data model is marked as finalized in the XML'() {
```
Comment (sdelamo) [inline ModelCatalogueCorePluginTestApp/test/functional/org/modelcatalogue/core/remoteTesting/FinalizedDataModelIsMarkedAsFinalizedInXMLSpec.groovy:29]: @Vendetta30 Could you annotate this test with: 

`@IgnoreIf({ !System.getProperty("downloadFilepath") })`

File: ModelCatalogueCorePluginTestApp/test/functional/org/modelcatalogue/core/remoteTesting/FinalizedDataModelIsMarkedAsFinalizedInXMLSpec.groovy [hunk 3]
```diff
@@ -42,5 +47,28 @@ class FinalizedDataModelIsMarkedAsFinalizedInXMLSpec extends GebSpec {
...
+        dataModelPage.exportXml()
+        Thread.sleep(2000)
+
+        file.eachFile(FileType.FILES) {
+            files.add(it.name)
+        }
+
+        then:
+        files.contains("Cancer_Model.mc.xml")
+        cancelFile.text.contains(dataModelName)
```
Comment (sdelamo) [inline ModelCatalogueCorePluginTestApp/test/functional/org/modelcatalogue/core/remoteTesting/FinalizedDataModelIsMarkedAsFinalizedInXMLSpec.groovy]: @Vendetta30 this should check that the status is Finalized in the XML. See attachment

napkin 7 02-04-18 10 01 24 am

**Review by sdelamo — CHANGES_REQUESTED** [review]

**Comment by vijayshukla30** [comment]
@sdelamo 
I did the changes as you suggested, please check and let me know.
https://github.com/MetadataConsulting/ModelCataloguePlugin/commit/7b37d03562a7dd9cbcd6b9901e7388fbb09da72b

**Comment by sdelamo** [comment]
Thanks @Vendetta30 I have organized the file deletion in the cleanup block and small changes. @davidmilward you can merge this PR.

**Comment by vijayshukla30** [comment]
@sdelamo 
Thanks, I will take care of it next time.

**Pull request merged**